libcyphal_demo/src/platform/linux/epoll_single_threaded_executor.hpp

```diff
@@ -0,0 +1,244 @@
+// This software is distributed under the terms of the MIT License.
+// Copyright (C) OpenCyphal Development Team  <opencyphal.org>
+// Copyright Amazon.com Inc. or its affiliates.
+// SPDX-License-Identifier: MIT
```

Comment:
These `platform/**` files are copy/paste from libcyphal's example as is (only root namespace has been changed).